docs/faq_tips_tricks.rst

```diff
@@ -0,0 +1,31 @@
+.. include:: links.rst
+
+FAQ, Tips, and Tricks
+------------------------------
+
+**Should I run quality control of my images before running ``FMRIPREP``?**
```

Comment:
Here's the rendered result: https://8229-53175327-gh.circle-artifacts.com/0/home/circleci/out/docs/faq_tips_tricks.html

The back-ticks don't seem to work in bold. Also, if we make this a sub-heading, then it will be linkable.

```suggestion
Should I run quality control of my images before running fMRIPrep?
------------------------------------------------------------------
```